src/main/scala/fs2/kafka/ConsumerSettings.scala

```diff
@@ -206,6 +209,16 @@ sealed abstract class ConsumerSettings[K, V] {
     */
   def withDefaultApiTimeout(defaultApiTimeout: FiniteDuration): ConsumerSettings[K, V]
 
+  /**
+    * Returns a new [[ConsumerSettings]] instance with read isolation level
+    * set to either "read_committed" (`true`) or "read_uncommitted" (`false`).
+    *
+    * {{{
+    * ConsumerConfig.ISOLATION_LEVEL_CONFIG
+    * }}}
+    */
+  def withReadCommittedIsolation(onlyReadCommitted: Boolean): ConsumerSettings[K, V]
```

Comment:
Wasn't sure if I should make this a `Boolean` or make a new type similar to `Acks`... thoughts?